recipes/tixi3/all/conanfile.py

```diff
@@ -0,0 +1,127 @@
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools import files
+from conan import ConanFile
+import os
+import textwrap
+
+required_conan_version = ">=1.45.0"
+
+
+class Tixi3Conan(ConanFile):
+    name = "tixi3"
```

Comment:
Please, update the recipe name to `tixi` following the project name. Also, you need to update the class name and folder name as well

Reply:
The name tixi3 is intentional to be able to use both tixi and tixi3 simultaneously. We did the same when creating conda packages for it, there is a tixi and a tixi3 package containing the tixi and tixi3 library.
https://anaconda.org/dlr-sc/repo?type=conda&label=main

Hence, I don't really like to rename this. This is similar to the libxml2 package, which is not called libxml.

Reply:
@rainman110 I don't get, what do you mean `tixi` project, besides `https://github.com/DLR-SC/tixi`? Are you considering running two different versions at same project?

Reply:
In our internal bigger build chain, we use both libraries simulateously to be able to update particular "cpacs" xml files. 

We have a geometry processor tigl, whose datamodel has changed a lot with the major version change from 2 to 3. To be able to update the data model and the geometry representation, we developed an update tool using both tigl and tigl3. It will be load tigl library and generate an aircraft geonetry. Tigl requires the tixi library. Then some transformations will be done and the geometry is passed to the tigl3 library, which requires tixi3. This is intentional, as our users need to transition from tigl to tigl3.

Long Story short: we need to be able to link both tixi and tixi3 simultaneously, hence the decision for the tixi3 package name.

From the technical point of view: The official tixi cmake config file called, tixi3-config.cmake with a tixi3 target:
https://github.com/DLR-SC/tixi/blob/master/src/CMakeLists.txt#L86

```cmake
install (FILES
    "${CMAKE_CURRENT_BINARY_DIR}/tixi3-config.cmake"
    "${CMAKE_CURRENT_BINARY_DIR}/tixi3-config-version.cmake"
    DESTINATION ${CONFIG_INSTALL_DIR})
```

It really does not make any sense to use the name "tixi" without a 3 for this package.

Reply:
@rainman110 Thank you for clarifying. In that case, makes sense keeping tixi3.